src/pages/index.js

```diff
@@ -20,88 +64,189 @@ const cardSection = new Section(
   ".gallery__cards"
 );
 
-// Render initial cards
-cardSection.renderItems();
+// Fetch API cards and render them inside the same section
+api.getInitialCards()
+.then((cards) => {
+  cards.forEach((cardData) => {
+    const cardElement = createCard(cardData); // Generate card using createCard
+    cardSection.addItem(cardElement); // Properly add to `.gallery__cards`
+    const likeBtn = document.querySelector('.card__like-button');
+  })
+  .catch(err=>{console.log('somethings wrong with getInitialCards', err)})
+});
```

Comment:
* There is an error in the console https://snipboard.io/OAZLW1.jpg 

The `.catch` block should be in line 76 in the promise chain

```js
// Fetch API cards and render them inside the same section
api
  .getInitialCards()
  .then((cards) => {
    cards.forEach((cardData) => {
      const cardElement = createCard(cardData); // Generate card using createCard
      cardSection.addItem(cardElement); // Properly add to `.gallery__cards`
    });
  })
  .catch((err) => {
    console.log("somethings wrong with getInitialCards", err);
  });

```